### ISSUE TITLE
Release 8.1

### ISSUE THREAD

**Issue by roigcarlo** [KratosMultiphysics/Kratos#7727]
Dear @KratosMultiphysics/all we are planning to make Release 9.0 by the end of the month (27/11/2020).

We have created a [milestone](https://github.com/KratosMultiphysics/Kratos/milestones) so its easier to track which branches are needed to be merged before the release. All PR should be merged before **16/11/2020** so we have a week to ensure everything works before the final release. 

Also bear in mind that this will be the last release with the current application structure, and for Release 10 we will introduce the new Research & Core application structure in the repository.

Thx all.

**Edit**:
Release will be on the 27, not the 28 (which is Saturday). Sorry for the mistake.

**Comment by rezanajianasl** [comment]
@roigcarlo I have been using and evaluating Kratos for a while and I would like to share my experience here, maybe it draws your attention. Since Februray, CI process (nightly build #7551) is failing and also I noticed that some tests (e.g. StructuralMechanicsApp tests) are failing even in the previous release (Release. 8). 

These behaviours leave an impression that Kratos is neither stable and nor reliable. I believe releasing unstable code is not a good practice. Also I think the test stage reflects the facts that the code is not well maintained and the code quality is not good. 
Please correct me if I am missing details or I am wrong.

**Comment by loumalouomega** [comment]
I like it more when we did minor releases. We have become Firefox:
![imagen](https://user-images.githubusercontent.com/19991680/98019497-1968d280-1e02-11eb-8e56-4438281f877c.png)

**Comment by roigcarlo** [comment]
Hi @rezanajianasl, first thx for being honest with us, we always wellcome feedback. 

I would like to clarify the situation so you don't get the wrong impression (and hopefully others as well).

While it is true that some nightly tests are failing (correct me here if I am wrong, but currently only 2 from the cosimulation application ans 4 for the ContactStructural are failing in the nightly suit), its important to mention that the scope of those tests is among other things to check features that are not currently available to final users or from applications in development that will directly not make it to the Release at all. Also, please notice that in any case, 6 from more than 2000+ tests failing can be hardly considered unreliable in my opinion.

Since our repo also hosts research code which is in rapid development this situatons are quite frequent. We try our best to disable such tests or at least mark them as expected failures (se X, S in the tests suits) if we detect that the feature they pretend to test is not actively developed, but of course we might oversee some. We are planing to make the difference between those parts much more clear with the separation of the applications that are core features and the ones that are being developed in the context of our project (See #5270)

In conclusion, nightly tests failing might look bad, but in general it does not mean that the code we are going to release has flaws or major issues that we are ware off.

Also in a similar line I would like to express that sometimes we cannot chose if we want to make a release or not, as we have some contractual obligations with different projects. While we acknowledge that not all features we aim for could make it to the deadline, we try our best to ensure that everything that we ship into a release is working.

**Comment by roigcarlo** [comment]
@loumalouomega probably it will end up being 8.1

**Comment by RiccardoRossi** [comment]
Dear @rezanajianasl, to add something to the previous comments, not all the applications in Kratos have the same level of maturity. We acknowledge this is only clear to the inner team and we are thinking of ways to make it clearer.

Our plan is to explicitly cathegorize applications in "core" and "research", implying implicitly for which ones we (@KratosMultiphysics/technical-committee ) can guarantee maturity and robustness and which ones are still under development and may eventually fail. 

We hope to be able to carry out this step soon, however it has some technical difficulties which we will need to solve first.

**Comment by rezanajianasl** [comment]
Dear @RiccardoRossi @roigcarlo 
thanks a lot for the comments and clarifications, I fully understand the points and constraints you mentioned. However as an external user/developer to Kratos, I would like to comment more here since I have been observing Kratos repo for a while. 

I agree that releasing software with some known bugs below a certain severity is not necessarily bad. However, these known glitches should not cause a failing unit test. Otherwise, after each unit test run, one (specially as an external to the team) will have to look into the 20 failed unit tests and check one-by-one if the failure was an acceptable one or not. This gets cumbersome, error-prone, and discards a huge part of the automation aspect of unit tests.

If we really have tests for acceptable, known bugs, I think we can leverage unit testing tool's disable/ignore feature (so they are not run by default, only on-demand). Why am I repeating myself is that it is really a long time that CI is failing (since the begining of 2020). Externals who are not aware of internal strategies and practices get an impression that something is wrong with the code. Therefore, maybe it is not a good fit for their needs and applications. Which might not be really true. 

I highly appreciate Kratos team for their great work and supporting users and the community.
Thanks for your consideration.

**Comment by RiccardoRossi** [comment]
@RezaNajian I believe you are right. We will also discuss this in @KratosMultiphysics/technical-committee , but i believe we have agreement on your point, so thank you.

**Comment by philbucher** [comment]
just to say that tonight for the first time since we moved the nightly build to the Github Actions it worked without errors :)

**Comment by AlejandroCornejo** [comment]
When this is goint to be finished? It seems that I have a bug in my App induced with the recent changes (only when running in Dynamic ?¿)

**Comment by oberbichler** [comment]
Maybe a second Badge/CI for the release branch would be enought.

E.g.
![image](https://user-images.githubusercontent.com/7000594/99186392-472e0f80-2750-11eb-9f49-dbe83ec44344.png)

**Comment by pooyan-dadvand** [comment]
@KratosMultiphysics/technical-committee, after revising and reflecting the comments, concluded that we are in lack of time to properly prepare and test a major release now before the christmas vacations. 

So we will only release a minor release which is scheduled also for the ExaQUte project which will be 8.1.0

**Comment by jginternational** [comment]
It will be cheered with beer anyway
🍻
